paper/src/main/java/com/denizenscript/denizen/paper/properties/PaperPlayerExtensions.java

```diff
@@ -26,6 +30,42 @@ public static void register() {
             return new ElementTag(object.getPlayerEntity().getAffectsSpawning());
         });
 
+        // <--[tag]
+        // @attribute <PlayerTag.client_options>
+        // @returns MapTag
+        // @group paper
+        // @Plugin Paper
+        // @description
+        // Returns the player's client options.
+        // The allow_server_listings & text_filtering_enabled options are only available in MC 1.19+.

```

Comment:
Needs to document what exactly does the map have - but are we sure a map is the best way to implement this? it'll probably mostly be used as `<[player].client_options.get[option]>`, might make more sense to do `<[player].client_option[<option>]>` or something?